flow_prompt/ai_models/gemini/gemini_model.py

```diff
@@ -102,11 +102,10 @@ def call(self, messages: t.List[dict], max_tokens: int, **kwargs) -> AIResponse:
         content = ""
         
         try:
-            if kwargs.get('stream'):
+            if not kwargs.get('stream'):
```

Comment:
why is it different?

Reply:
I mixed up the if statements. It was streaming when shouldn't and vice versa